### PR TITLE
Improve lsm_loss description and variables names

### PR DESCRIPTION
**Pull request by denkle**

### CODE REVIEW

**Comment by belsten** [comment]
Looks great! There is a typo in "sparse coding" in line 322 and 354.

**Comment by denkle** [comment]
@belsten thanks for spotting! Fixed it.

File: sparsecoding/inference.py [hunk 1]
```diff
@@ -347,15 +376,16 @@ def infer(self, data, dictionary):
             
             # Set coefficients to zero before doing repeating the inference with new lambdas
             coefficients=torch.zeros(batch_size, n_basis, requires_grad=True)
```
Comment (belsten) [inline sparsecoding/inference.py]: These coefficients should be cast to to dictionary device defined above.

Comment (denkle) [reply sparsecoding/inference.py]: Yeap, updated

**Pull request merged**